modules/ROOT/partials/configuration/custom_elements.adoc

```diff
@@ -5,9 +5,9 @@ This option enables you to specify non-HTML elements for the editor.
 
 This way you can handle non-HTML elements inside an HTML editor. You can prefix the element names with a `+~+` if you want the element to behave as a `+span+` element and not a `+div+` element.
 
-Type : `+String+`
+*Type:* `+String+`
 
-Default Value : `+div+` elements
+*Default value:* `+'div'+` elements
```

Comment:
Unrelated to the changes but noticed it and wondered if this default value should even be present for the `custom_elements` option?

Reply:
Agreed this should not be here.